runtimevar/runtimevar.go

```diff
@@ -510,20 +512,42 @@ func DecryptDecode(ctx context.Context, k *secrets.Keeper, post Decode) Decode {
 //   - "jsonmap": Returns a JSON decoder for a map[string]interface{};
 //       Snapshot.Value will be of type *map[string]interface{}.
 //   - "string": Returns StringDecoder; Snapshot.Value will be of type string.
-func DecoderByName(decoderName string, dflt *Decoder) (*Decoder, error) {
+// It also supports using "decrypt/<decoderName>" (or "decrypt" for default
+// decoder) to decrypt the data before decoding. It uses the secrets package to
+// open a keeper by the URL string stored in a envrionment variable
+// "RUNTIMEVAR_KEEPER_URL". See https://godoc.org/gocloud.dev/secrets#OpenKeeper
+// for more details.
+func DecoderByName(ctx context.Context, decoderName string, dflt *Decoder) (*Decoder, error) {
```

Comment:
BTW, your change *is* a breaking change (as you noted); it's not failing the Travis checks because I accidentally checked in something from testing (see #1612 for me fixing it; once I merge that, if you merge it into this PR, the checks would fail unless you have a git commit with `BREAKING_CHANGE_OK` in it).

This is fine, especially because we don't intend external users to use this function (it is exported for user by drivers). WDYT about changing it to use the same trick we used for `New` above, i.e., having it exported as a variable instead of a function? The functionality is the same but it hides the signature in godoc.